apps/picsa-tools/option-tool/src/app/material.module.ts

```diff
@@ -1,10 +1,17 @@
 import { NgModule } from '@angular/core';
 
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { MatButtonModule } from '@angular/material/button';
 import { MatDialogModule } from '@angular/material/dialog';
 import { MatTableModule } from '@angular/material/table';
 
-const COMPONENTS = [MatButtonModule, MatDialogModule, MatTableModule];
+const COMPONENTS = [
+  MatButtonModule,
+  MatDialogModule,
+  MatTableModule,
+  FormsModule,
+  ReactiveFormsModule,
```

Comment:
comment(blocking)
I'm glad to see you figured out how to use angular forms with the use of the FormsModule and ReactiveFormsModule. 
As these imports don't strictly relate to the material components I think we can probably move to the main `app.module` instead, which will still inherit in your template